src/features/content-insights/__tests__/ContentInsightsSummary.test.tsx

```diff
@@ -38,9 +41,28 @@ describe('features/content-insights/ContentInsightsSummary', () => {
             expect(screen.queryByLabelText(localize(messages.previewGraphLabel.id))).toBeNull();
         });
 
-        test('should render correctly when isLoading is false', () => {
+        test('should render correctly when isLoading is false and error is null', () => {
+            mockError.mockReturnValueOnce(null);
+            getWrapper();
+
+            expect(screen.queryByTestId('ContentAnalyticsErrorState')).toBeNull();
+            expect(screen.queryByTestId('GraphCardGhostState')).toBeNull();
+            expect(screen.getByLabelText(localize(messages.previewGraphLabel.id))).toBeVisible();
+        });
+        test('should show the error state when isError is true', () => {
```

Comment:
Should this say `error` instead of `isError`? Seems like isError was updated/removed from the props?